arraykit.c

```diff
@@ -61,6 +63,8 @@
         fprintf(stderr, #msg);  \
     _AK_DEBUG_END()
 
+# define isnanj(val) (isnan(val.real) || isnan(val.imag))
+
```

Comment:
I'm not convinced that this deserves to be a macro. It doesn't really save us any space, the name is a bit misleading, and it hides a potentially nasty double-evaluation.

Maybe just replace its four uses below with the expanded version?

```suggestion
```